tests/utils.py

```diff
@@ -65,10 +66,13 @@ def __init__(self,
                  env_dict: Optional[Dict[str, str]] = None,
                  auto_port: bool = True,
                  max_wait_seconds: Optional[float] = None) -> None:
-        if not model.startswith("/"):
-            # download the model if it's not a local path
-            # to exclude the model download time from the server start time
-            snapshot_download(model)
+
+        # Prepare weights so if the model is not local, it will be downloaded.
+        # We do this in the constructor to exclude the model download time
+        # from the server start time.
+        loader = get_model_loader(LoadConfig())
```

Comment:
is this `LoadConfig()` necessary?

you can directly use the default loader I think